newrelic/hooks/framework_django.py

```diff
@@ -1222,3 +1235,13 @@ def instrument_django_core_handlers_exception(module):
         module.handle_uncaught_exception = (
                 wrap_handle_uncaught_exception(
                 module.handle_uncaught_exception))
+
+
+def instrument_django_core_handlers_asgi(module):
+    import django
+
+    framework = ('Django', django.get_version())
+
```

Comment:
Are these lines actually necessary? Looks like they're unused.

Reply:
They're used, an assertion is missing. Nice catch! 😄